main/guides/getting-started/README.md

```diff
@@ -68,7 +68,7 @@ go version # Version 1.20.3 or higher
 ## Install Node.js
 
 Download Node.js from [Node.js](https://nodejs.org/) and follow the instructions for your platform.
-We recommend installing the LTS version of node 18.
+We support Long Term Support (LTS) versions of node; currently, version 18.
```

Comment:
while you're here, "Node" should be capitalized. I think we should link to the Node docs on releases. And we can drop the version number from the copy here as it's in the shell comment right below. The "Current" release is likely to work but is not officially supported.

```suggestion
We support the Long Term Support (LTS) releases of Node, both [Active and Maintenance](https://nodejs.org/en/about/previous-releases).
```